docs/pages/table.js

```diff
@@ -145,90 +75,1666 @@ card(
+    );
+
+    const BaseRow = ({ campaign, status, budget, scope }) => (
+      <Table.Row>
+        <Table.HeaderCell>
+          <Box display="visuallyHidden">
+            <Text weight="bold">Open/Close row</Text>
+          </Box>
+        </Table.HeaderCell>
+        <Table.Cell>
+          <Text>{campaign}</Text>
+        </Table.Cell>
+        <Table.Cell>
+          <Text>{status}</Text>
+        </Table.Cell>
+        <Table.Cell>
+          <Text>{budget}</Text>
+        </Table.Cell>
+        <Table.Cell>
+          <Text>{scope}</Text>
+        </Table.Cell>
+      </Table.Row>
+    );
+
+    const RowExpandable = ({ campaign, status, empty, budget, scope }) => (
+      <Table.RowExpandable
+        accessibilityExpandLabel="Expand"
+        accessibilityCollapseLabel="Collapse"
+        id="row1"
+        onExpand={() => {}}
+        expandedContents={
+          empty ? (
+            <Text>No metrics available. This campaign hasn't started yet.</Text>
+          ) : (
+            <ExpandedContents />
+          )
+        }
+      >
+        <Table.Cell>
+          <Text>{campaign}</Text>
+        </Table.Cell>
+        <Table.Cell>
+          <Text>{status}</Text>
+        </Table.Cell>
+        <Table.Cell>
+          <Text>{budget}</Text>
+        </Table.Cell>
+        <Table.Cell>
+          <Text>{scope}</Text>
+        </Table.Cell>
+      </Table.RowExpandable>
+    );
+
+    const tableID = "Another example of a 'don't' do for table content";
+
+    return (
+      <Table accessibilityLabel={tableID}>
+        <HeaderRow />
+        <Table.Body>
+          <RowExpandable campaign="December '21" status="Active" budget="$100,000" scope="Global" />
+          <RowExpandable
+            campaign="January '22"
+            status="Draft"
+            budget="$50,000"
+            scope="Japan, Germany, Canada, Spain, Mexico, Thailand, Italy"
+            empty
+          />
+          <RowExpandable
+            campaign="February '22"
+            status="Draft"
+            budget="$50,000"
+            scope="Japan, Germany, Canada"
+            empty
+          />
+        </Table.Body>
+      </Table>
+    );
+  }
+
+  return <Example />;
+}
+    `}
+          />
+        </MainSection.Subsection>
+      </MainSection>
+      <MainSection
+        name="Localization"
+        description={`Be sure to localize \`text\` and \`accessibilityLabel\`.
+
+Note that localization can lengthen text by 20 to 30 percent; follow our guidelines on concise content and headings to account for localization.
+
+Wrap important table content instead of truncating. Use truncation only for secondary content, and include a tooltip to show the full text on hover.
+`}
+      />
+      <MainSection name="Accessibility">
+        <MainSection.Subsection
+          title="Labels"
+          description={`
+Use  \`accessibilityLabel\` to properly announce the content of the table. For example, use “Ad Groups Table” instead of “Table”.
```

Comment:
This seems counterintuitive to the sentence below. Maybe: "For example, use “Campaign Status Information” instead of “Info”.

Reply:
DONE